content/20.zksync-protocol/05.gateway/00.overview.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Overview
+description:
+---
+
+ZKsync Gateway is an **optional settlement aggregation layer** for ZK chains, including both rollups and validiums.
+It is purpose-built to enhance **interoperability**, provide **proof aggregation**, and offer **cost efficiencies** for
+chains that choose to settle on top of it.
+
+As a settlement aggregation layer between ZK chains and Ethereum, ZKsync Gateway collects proofs from multiple chains and submits them to Ethereum
+for final verification. This reduces gas costs for proof verification while preserving Ethereum as the root of trust.
+
+Assets on Gateway-settling chains remain **locked on Ethereum**, and all L1-to-L2 interactions—such as deposits—are still executed from Ethereum,
+not through Gateway.
+
+## Key Benefits
+
+- **Interoperability**: Enables faster communication between ZK chains settling on Gateway, without affecting direct L1–L2 connectivity from Ethereum.
+- **Proof Aggregation**: Reduces costs for users and chains by aggregating multiple proofs into a single proof submitted to Ethereum.
+- **Cost Efficiency**: Offers stable pricing for proof verification compared to direct settlement on Ethereum,
+  while still maintaining Ethereum-based asset security.
+
+## Governance
+
+ZKsync Gateway was **whitelisted as a settlement layer** in the [ZIP-10 governance proposal](https://www.tally.xyz/gov/zksync/proposal/97689115420129047109255183628089175185608660755000395855946331923921270505453?govId=eip155:324:0x76705327e682F2d96943280D99464Ab61219e34f).
+This proposal allows chain operators to select ZKsync Gateway as their settlement layer in place of Ethereum.
+
+Migration to ZKsync Gateway is **optional**, and chains initially launch with Ethereum settlement.
+If desired, operators may later configure their chain to use Gateway as the settlement layer.
+Switching between Ethereum and Gateway is supported at the protocol level, but a chain must first be deployed with Ethereum as its origin settlement layer.
+
+::callout{icon="i-heroicons-exclamation-triangle" color="amber"}
+Learn more about [migrating an existing chain to Gateway in the ZK Stack documentation](../../zk-stack/running/gateway-settlement-layer).
+::
+
+## Gateway Network Details
+
+| Environment | Chain ID | Gas Token | Explorer |
+|-------------|----------|-----------|----------|
+| Testnet     | 506      | $ZK       | [Explorer](https://gateway.explorer.zksync.io) |
```

Comment:
same links

Reply:
yeah, these are placeholders as we don't have the domain configured yet